src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -54,30 +54,52 @@ describe('lib/viewers/image/ImageViewer', () => {
     });
 
     describe('destroy()', () => {
-        test('should remove the zoom event listener', () => {
+        beforeEach(() => {
+            jest.spyOn(image, 'removeListener');
+        });
+
+        test('should remove the zoom event listener if enableAnnotationsImageDiscoverability is true', () => {
             image.options.enableAnnotationsImageDiscoverability = true;
-            image.removeListener = jest.fn();
 
             image.destroy();
 
             expect(image.removeListener).toBeCalledWith('zoom', expect.any(Function));
         });
+
+        test('should not remove the zoom event listener if enableAnnotationsImageDiscoverability is false', () => {
```

Comment:
Could combine these two into 1 with a `test.each` and assert with `toHaveBeenCalledTimes`